internal/command/operator.go

```diff
@@ -298,6 +302,26 @@ Note: If --auto-registry-credentials and --registry-credentials-path are unset,
 			if stdout {
 				applier = kubernetes.NewStdOutApplier()
 			} else {
+				// before starting the application of the installation instance,
+				// we can check if the installation CRD is present
+				kubeCfg, err := kubernetes.NewConfig(kubeConfig)
+				if err != nil {
+					return err
+				}
+
+				installationClient, err := operator.NewInstallationClient(kubeCfg)
+				if err != nil {
+					return err
+				}
+
+				_, err = installationClient.Status(ctx)
```

Comment:
This checks whether there is an installation in cluster and errors out if there isn't- so it does not allow creating a new installation